Dropdown/Dropdown.js

```diff
@@ -156,7 +166,9 @@ const DropdownBase = kind({
 	},
 
 	computed: {
+		className: ({open, styler}) => styler.append({listOpen: open, listClosed: !open}),
```

Comment:
`listClosed` might not used anywhere. Could we remove it?
And in my opinion, it would be just fine to use `open` instead of `listOpen`.